packages/modules/src/modules/banners/charityAppeal/CharityAppealBannerReminderSignedOut.tsx

```diff
@@ -0,0 +1,325 @@
+import React from 'react';
+import { css, ThemeProvider } from '@emotion/react';
+import { Button, buttonReaderRevenueBrand } from '@guardian/src-button';
+import { textSans } from '@guardian/src-foundations/typography';
+import { neutral, space } from '@guardian/src-foundations';
+import { Columns, Column, Hide } from '@guardian/src-layout';
+import { from } from '@guardian/src-foundations/mq';
+import { TextInput } from '@guardian/src-text-input';
+import { SvgCheckmark } from '@guardian/src-icons';
+import { BannerEnrichedReminderCta } from '../common/types';
+import { ensureHasPreposition, ReminderStatus } from '../../utils/reminders';
+import { useContributionsReminderEmailForm } from '../../../hooks/useContributionsReminderEmailForm';
+import { ErrorCopy, InfoCopy, ThankYou } from '../../shared/Reminders';
+
+const dfltForeColor = neutral[100];
```

Comment:
From what I can tell this variable isn't needed, it doesn't seem to be conditionally passed into the css functions